### PR TITLE
Create Karma config file

### PR DESCRIPTION
**Pull request by alexpattison**

### CODE REVIEW

**Comment by jbg46209** [comment]
:+1:

File: src/client/services/services.js [hunk 1]
```diff
@@ -92,7 +92,7 @@ app.factory('RestaurantAndRoute', ['$http', function($http) {
         markers.push(marker);
 
         //Display an info window when marker is clicked
-        let infoWindow = createInfoWindow(place);
+        let infoWindow = Window(place);
```
Comment (cchrispy) [inline src/client/services/services.js:95]: is Window a function?

Comment (alexpattison) [reply src/client/services/services.js:95]: This actually came in from the rebase, not sure why it's in my diff

Comment (cchrispy) [reply src/client/services/services.js:95]: so it's fine to leave it as Window instead of createInfoWindow?

**Pull request merged**